examples/with_plugs.py

```diff
@@ -74,13 +74,17 @@ class PingDnsB(PingPlug):
 # passed into the phase so each phase has a unique name.
 @htf.PhaseOptions(name='Ping-{pinger.host}-{count}')
 @htf.plug(pinger=PingPlug.placeholder)
-@htf.measures('total_time_{pinger.host}_{count}', 'retcode')
-def test_ping(test, pinger, count):
+@htf.measures(
+    'total_time_{pinger.host}_{count}',
+    htf.Measurement('retcode').equals('{expected_retcode}', type=int)
+)
+def test_ping(test, pinger, count, expected_retcode):
   """This tests that we can ping a host.
 
   The plug, pinger, is expected to be replaced at test creation time, so the
   placeholder property was used instead of the class directly.
   """
+  del expected_retcode  # Not used in the phase, only used by a measurement
```

Comment:
I'm making it clear that it has to be an input argument but doesn't have to be used in the phase